### PR TITLE
Adds telemetry handling, writes multiple bytes

### PR DESCRIPTION
**Pull request by pkubanek**
Adds the ability to send multiple bytes (command 20) and telemetry FIFO writing routines (commands 30-33).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,25 +16,70 @@ reading out data from Modbus devices. The design includes:
...
-* write something (transformed part of output array) to telemetry FIFO
+## Supported instructions
+
+| Instruction | Description                                             |
+| ----------- | ------------------------------------------------------- |
+| 0           | Stop executing and wait for a new instructions in FIFO. |
+| 1           | Write the next byte to the port.                        |
+| 2           | Wait for given ms.                                      |
+| 3           | Read from the port and store data into output array.    |
+| 4           | Loop (jump to some instruction, usually 0 to repeat the sequence; with  looping, a check for commanding FIFO will be provided, and if that contains new instructions, instruction stack will be replaced with those new instructions). |
```
Comment (teweitsai) [inline README.md:27]: " with  looping" -->  "with looping"

File: README.md [hunk 2]
```diff
@@ -16,25 +16,70 @@ reading out data from Modbus devices. The design includes:
...
+0. 23
+1. 20 4 1 58 80 33
+2. 2 100
+3. 3 16
+4. 5
+5. 31 11 2
+6. 33 22 4
+7. 32 67 12
+8. 2 500
+9. 4
```
Comment (teweitsai) [inline README.md:68]: Please consider to add more explanation for this part.

File: README.md [hunk 3]
```diff
@@ -16,25 +16,70 @@ reading out data from Modbus devices. The design includes:
...
+
+Assuming device response is (CRC is 0x6676):
+
+1 58 0x11 0x12 0x13 0x14 0x15 0x16 0x17 0x18 0x19 0x20 0x21 0x22 0x23 0x24 0x76 0x66
```
Comment (teweitsai) [inline README.md:72]: Please consider to add more explanation for this part.

**Review by teweitsai — APPROVED** [review]
The update looks good to me.

File: README.md (hunk 1 above)
Comment (pkubanek) [reply README.md:27]: Fixed, thanks

File: README.md (hunk 2 above)
Comment (pkubanek) [reply README.md]: Most is documented in paragraph above this text. Added what 0 means (sending length of data). Also added unite test to cover this.

File: README.md (hunk 3 above)
Comment (pkubanek) [reply README.md:74]: Those are raw data from device. I am not sure how much I can add.

**Pull request merged**